lib/realtime/database.ex

```diff
@@ -133,6 +126,14 @@ defmodule Realtime.Database do
     end)
   end
 
+  def connect(tenant, application_name, pool, backoff \\ :stop) do
+    tenant
+    |> then(&Realtime.PostgresCdc.filter_settings(@cdc, &1.extensions))
+    |> then(&Realtime.Database.from_settings(&1, application_name, backoff))
+    |> then(&%{&1 | pool: pool})
```

Comment:
This is defo still confusing because some pool sizes we set in the tenant record and some we override here?

Reply:
yeah we need to potentially refactor more to have a better way to handle this but this at least already provides a better escape hatch and more concise code